packages/bpk-component-switch/src/BpkSwitch.module.scss

```diff
@@ -18,13 +18,15 @@
 
 @import '~bpk-mixins/index';
 
+$bpk-spacing-v2: true;
+
 $primary-on-color: $bpk-color-sky-blue;
 $event-on-color: $bpk-color-abisko;
 $off-color: $bpk-color-sky-gray-tint-06;
 $handle-color: $bpk-color-white;
 
-$height: $bpk-spacing-lg;
-$width: $bpk-spacing-base * 2;
+$height: bpk-spacing-xl();
+$width: bpk-spacing-lg() * 2;
```

Comment:
I'm not quite sure about these changes. The switch just doesn't quite look right here 🤔

Reply:
It it is going to be taller, I think it needs to be wider too to maintain proportions. wdyt?

Reply:
I was wondering about this too and it not looking too right. I tried to make it a bit wider but would possibly mean making the track wider too which then made it look a bit weirder 🤔

Reply:
Might be similar to our checkbox and radio components we make this one use the sizes it was before with the magic of CSS maths just to make it look balanced otherwise either way it might look a bit off, wdyt? Will also check in with design on this too